Changelog.md

```diff
@@ -1,5 +1,18 @@
 # Changelog for the `parameterized-utils` package
 
+## 2.1.0 -- *2020 May 08*
+
+  * Added `plusAssoc` to the `NatRepr` module to produce `+` associativity evidence.
+  * Changed the `HashTable` module to use the Basic instead of Linear
+    implementation strategy.
```

Comment:
I think it would make sense to say `Basic` instead of `Cuckoo` here - no released version used `Linear`

Reply:
Thanks for  catching this!